docker-compose.yml

```diff
@@ -1,29 +1,35 @@
-version: '0.1'
-
+version: '3.7'
```

Comment:
Updated the version to a valid version.  Otherwise "docker-compose up -d" would not work and say invalid version.